assets/base.css

```diff
@@ -2533,8 +2534,18 @@ details-disclosure > details {
   z-index: -1;
 }
 
+.global-media-settings--no-shadow {
+  overflow: hidden !important;
+}
+
+.global-media-settings--no-shadow:after {
+  content: none;
+}
+
 .global-media-settings img,
-.global-media-settings iframe {
+.global-media-settings iframe,
+.global-media-settings model-viewer,
+.global-media-settings video {
```

Comment:
Didn't we want this to not be applied in the modal?
[video](https://screenshot.click/22-01-lxzd8-d40gg.mp4)